plugins/flytekit-kf-mpi/flytekitplugins/kfmpi/task.py

```diff
@@ -133,5 +133,62 @@ def get_custom(self, settings: SerializationSettings) -> Dict[str, Any]:
         return MessageToDict(job.to_flyte_idl())
 
 
+@dataclass
+class HorovodJob(object):
+    slots: int
+    num_launcher_replicas: int = 1
+    num_workers: int = 1
+
+
+class HorovodFunctionTask(MPIFunctionTask):
+    """
+    For more info, check out https://github.com/horovod/horovod
+    """
+
+    # Customize your setup here. Please ensure the cmd, path, volume, etc are available in the pod.
+    ssh_command = "/usr/sbin/sshd -De -f /home/jobuser/.sshd_config"
```

Comment:
Is it possible to get the user from some config instead of having to hardcode it to `jobuser`? maybe use `run_as` in Flyte?

Reply:
But the config cannot always infer ssh_command? Because user might locate their `sshd_config` in various places